src/elements/SelectField/SelectField.styl

```diff
@@ -94,6 +94,11 @@
   sizes('font-size')
   font-weight $weight-bold
 
+  height $height-control
+  width $width-control
+  box-sizing $box-sizing-control
+
+
```

Comment:
Please ensure that across all files, only one blank line separates adjacent code blocks.